### DISCUSSION TITLE
Is there any particular reason for why the translation json files are in the public folder?

### DISCUSSION THREAD

**Discussion by Axedyson** [i18next/next-i18next#1851] [Q&A] [answered]
What is the benefit of having the translation json files in `public/locales` folder vs just `./locales` folder? I.e. what is the benefit of making translation files available for everyone to access?

**Comment by isaachinman** [comment, marked as answer]
The reason is pretty straightforward – so we can download them on the client side if needed. This used less these days, as `getStaticProps` and `getServerSideProps` handle the chunking/management of assets for us.

It used to be the case that incoming namespace dependencies needed to be managed manually in `getInitialProps`.

**Reply by Axedyson** [reply]
> so we can download them on the client side if needed

When would such a scenario occur? I guess you're referring to this:
https://github.com/i18next/i18next-http-backend/tree/master/example/next

But would it be completely okay to just leave the files out of the public folder since I don't think I'll need to download translations on the client side?

**Reply by isaachinman** [reply]
Yes, of course.
